src/pytest_netdut/wrappers.py

```diff
@@ -34,36 +34,84 @@ def _splitcmds(cmds):
     return cmds
 
 
-def _eos_to_mos_translator(cmds):
-    translations = [
+class Translator:
+    """Base class which implements translation functions for hiding syntax
+    and result differences between devices with different operating systems.
+
+    Override the standardize_key function to implement result processing.
+
+    Args:
+        config_patterns (iterable[regex pattern]): List of regex patterns for CLI translations.
+    """
+
+    config_patterns = []
+
+    def standardize_key(self, k):
+        """Returns a standardized key.
```

Comment:
It's not clear from this docstring what this key is for. I think this is for translating json results?